tests/wpt/metadata/url/urlsearchparams-sort.any.js.ini

```diff
@@ -1,105 +1,20 @@
 [urlsearchparams-sort.any.html]
-  [Parse and sort: a🌈&a💩]
-    expected: FAIL
-
-  [URL parse and sort: z=z&a=a&z=y&a=b&z=x&a=c&z=w&a=d&z=v&a=e&z=u&a=f&z=t&a=g]
-    expected: FAIL
-
-  [URL parse and sort: bbb&bb&aaa&aa=x&aa=y]
-    expected: FAIL
-
-  [URL parse and sort: é&e�&é]
-    expected: FAIL
-
   [Parse and sort: ﬃ&🌈]
```

Comment:
I'm still very confused about this test case.

https://github.com/web-platform-tests/wpt/blob/1d3a454b4af3f3d70206b53a6dce49ae0ddcc7a9/url/urlsearchparams-sort.any.js#L10-L13

The comment said `🌈 > code point, but < code unit because two code units`.
If I understand correctly, using `string.chars()`, we can see `code points` for the string; `🌈` is `[240, 159, 140, 136]` and `ﬃ` is `[239, 172, 131]`. Also, we can know `code units` via `string.encode_utf16().count()`; `🌈` is `2` and `ﬃ` is `1`. So I think `🌈` should be greater than `ﬃ` in both code points and code units?